mirrord/cli/src/db_branches.rs

```diff
@@ -0,0 +1,176 @@
+use kube::{
+    Api, Client,
+    api::{DeleteParams, ListParams},
+};
+use mirrord_operator::crd::mysql_branching::MysqlBranchDatabase;
+use prettytable::{Table, row};
+
+use crate::{
+    CliResult,
+    config::{DbBranchesArgs, DbBranchesCommand},
+};
+
+pub async fn db_branches_command(args: DbBranchesArgs) -> CliResult<()> {
+    match &args.command {
+        DbBranchesCommand::Status { names } => status_command(&args, names.clone()).await,
+        DbBranchesCommand::Destroy { all, names } => {
+            destroy_command(&args, *all, names.clone()).await
+        }
```

Comment:
Can `status_command()` and `destroy_command()` take a reference of `names` so we don't need to clone? It seems they are only used for resource filtering.